src/domain/wallets/index.types.d.ts

```diff
@@ -124,14 +124,29 @@ type WalletTransactionHistoryWithPending = {
   readonly transactions: WalletTransaction[]
 }
 
+type WalletName = string & { readonly brand: unique symbol }
```

Comment:
is this used?

Reply:
probably comes from this PR: https://github.com/GaloyMoney/galoy/pull/835. will remove it from now